_data/retail.yml

```
@@ -325,24 +386,36 @@ websites:
     url: https://www.pccasegear.com/
     twitter: pccasegear
     img: pccasegear.png
+    regions:
+      - au
 
   - name: Purse
     url: https://purse.io/
     img: purse.png
     tfa:
       - totp
     doc: https://support.purse.io/account/2-factor-authentication-2fa
+    regions:
+      - ca
+      - gb
+      - jp
+      - us
 
   - name: Rakuten
     url: https://www.rakuten.com/
     twitter: AskRakutenUS
     img: rakuten.png
+    regions:
+      - us
+      - jp
```

```suggestion
    img: rakuten.png
```
Rakuten has a [large international presence](https://global.rakuten.com/corp/about/company/).